### PR TITLE
conprof: update download readme

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Issue Number: close #xxx

### CODE REVIEW

**Comment by crazycs520** [comment]
@baurine PTAL

File: component/conprof/http/api.go [hunk 1]
```diff
@@ -356,7 +356,7 @@ func queryAndDownload(c *gin.Context) error {
 }
 
 const downloadReadme = `
-To review the CPU profiling or heap profiling result interactively:
+To review the profile data which file name suffix is `.proto` interactively:
```
Comment (baurine) [inline component/conprof/http/api.go:359]: Not sure whether the nested back-quote works.

Comment (baurine) [reply component/conprof/http/api.go:359]: And it is better to use `that` or `whose` to replace `which`, we can ask help from i18n.

**Review by baurine — APPROVED** [review]

**Comment by crazycs520** [comment]
@breeswish PTAL

**Review by breeswish — APPROVED** [review]

**Pull request merged**